src/components/ReportActionItem/IOUPreview.js

```diff
@@ -60,9 +60,15 @@ const propTypes = {
         /** Email address of the manager in this iou report */
         managerEmail: PropTypes.string,
 
+        /** Account ID of the manager in this iou report */
+        managerID: PropTypes.number,
```

Comment:
I think it's weird that is `managerID` instead of `managerAccountID`, it differs from what we have for owner

Reply:
Yeahhhhhhhh it's quite weird, but that's what it's like in the database :D (who designed that? NOT ME)